week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var perstudent = Math.round((100 * numberOfStudents) / total);
+var permentors = Math.round((100 * numberOfMentors) / total);
+console.log(perstudent);
+console.log(permentors);
```

Comment:
Nice! 
It's important to remember about naming your variables well. As a programmer, your code will always be read by other people so making variable and function names readable and clear is crucial - makes understanding of the code much easier :) We'd like you to learn that as early as possible, that's why we always point it out!

Reply:
@martynamaron Thanks.